packages/expo-modules-core/README.md

```diff
@@ -24,6 +24,75 @@ Run `npx pod-install` after installing the npm package.
 
 No additional set up necessary.
 
+# Importing native dependencies - autolinking
+
+React Native libraries come with platform-specific (native) code. This native code has to be linked into the project or
+configured properly. These actions require some modifications to the native project files. One of the steps that have
+to be done with the native configuration is to enable autolinking mechanism that takes care of including any supported
+module's native code into the project. Below we preset the configuration that should be manually applied to the
+project's native files.
+
+### iOS
+
+> Caution! After successful file modification you have to run `pod install` again.
+
+```ruby
+# Podfile
+
+require File.join(`node --print "require.resolve('react-native/package.json')"`, "../scripts/react_native_pods")
+require File.join(`node --print "require.resolve('@react-native-community/cli-platform-ios/package.json')"`, "../native_modules")
+require File.join(`node --print "require.resolve('@unimodules/react-native-adapter/package.json')"`, "../scripts/autolinking")
+
+#  ...
+
+target "TargetName" do
+  use_unimodules!
+  config = use_native_modules!
+  use_react_native!(:path => config["reactNativePath"])
+
+  # ...
+end
+```
+
+### Android
+
+```groovy
+// app/build.gradle
+
+apply from: new File(["node", "--print", "require.resolve('react-native-unimodules/package.json')"].execute().text.trim(), "../gradle.groovy")
+apply from: new File(["node", "--print", "require.resolve('react-native/package.json')"].execute().text.trim(), "../react.gradle")
+apply from: new File(["node", "--print", "require.resolve('expo-updates/package.json')"].execute().text.trim(), "../scripts/create-manifest-android.gradle")
```

Comment:
we'll need to update scripts for detecting expo-updates installation to reflect this. also will need to update expo-updates installation instructions: https://docs.expo.dev/bare/installing-updates/